### PR TITLE
Add the SLAlpha4 Release Note

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the SLAlpha4 release note.
> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -48,5 +48,5 @@ exclude: ["downloads/RELEASE_NOTE.md","vendor","0.991/","v1-2","v1-1","v1-0","v0
 #   - vendor/cache/
 #   - vendor/gems/
 #   - vendor/ruby/
-dist_server: https://dist.ballerina.io
+dist_server: https://dist-dev.ballerina.io
```
Comment (keizer619) [inline _config.yml:51]: Shall we revert these changes
@praneesha

**Comment by praneesha** [comment]
Closing since this is invalid.
